solution/src/main/kotlin/com/cosmotech/solution/utils/SolutionUtils.kt

```diff
@@ -7,7 +7,8 @@ import com.cosmotech.solution.domain.RunTemplateHandlerId
 fun getCloudPath(
     organizationId: String,
     solutionId: String,
-    handlerId: RunTemplateHandlerId
+    runTemplateId: String,
+    handlerId: RunTemplateHandlerId,
 ): String {
-  return "${organizationId}/${solutionId}/${handlerId}.zip"
+  return "${organizationId}/${solutionId}/${runTemplateId}/${handlerId}.zip"
```

Comment:
Not sure how this is handled later on, but to be consistent with what is done in the [file upload](https://github.com/Cosmo-Tech/cosmotech-api/blob/runtemplatezip_envvar/solution/src/main/kotlin/com/cosmotech/solution/azure/SolutionServiceImpl.kt#L402-L406), maybe this path here needs to be sanitized as well.

Reply:
yes thank you I will add this